hubblestack/extmods/fdg/grep.py

```diff
@@ -24,23 +24,29 @@ def file(path, pattern, grep_args=None, format_chained=True, chained=None, chain
     substitute the chained value.) If you want to avoid having to escape curly braces,
     set ``format_chained=False``.
 
+    chained_status
+        Status returned by the chained method.
+
     The first return value (status) will be True if the pattern is found, and
     False othewise. The second argument will be the output of the ``grep``
     command.
 
     ``grep_args`` can be used to pass in arguments to grep.
     """
     if format_chained:
-        pattern = pattern.format(chained)
-        path = path.format(chained)
+        if chained_status:
```

Comment:
Can you please remove these? This is actually a logic change -- I anticipate it's due to the fact that we're not using those arguments. But that's not necessarily a problem -- because fdg functions have a shared interface, then many of the functions may not use one or more of the parameters in that shared interface.

I want to keep logic changes out of lint fixes as much as possible.

(Also please remove all of these similar additions, I'm not going to leave a comment for each one.)

Reply:
I will clean them up, the `chained_status` argument was added afterwards and I thought this was the intended use for it. Even so, I agree that it make more sense to keep the logic changes out of the lint fixes and perhaps submit a different PR at some point

Reply:
If I remember correctly (relying on my memory is dangerous!) the chained_status was just there in case a module wanted the ability to change something about its logic based on the true/false status of the previous item in the chain.

It may be that we only want to `format_chained` if the `chained_status` was True, but I just don't want to make that decision in a lint fix, as discussed. So I think leaving it out is the correct move for now. Thanks for making that change!